server/game/cards/06-CotE/CourtlyChallenger.js

```diff
@@ -0,0 +1,51 @@
+const DrawCard = require('../../drawcard.js');
+const { Players, CardTypes } = require('../../Constants');
+
+class CourtlyChallenger extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            effect: [
+                ability.effects.delayedEffect({
+                    when: {
+                        afterDuel: (event) => event.winner && event.winner === this
+                    },
+                    message: '{0} is honored due to winning a duel',
+                    gameAction: ability.actions.honor(context => ({ target: context.source }))
+                })
+            ]
+        });
+        this.persistentEffect({
+            effect: [
+                ability.effects.delayedEffect({
+                    when: {
+                        afterDuel: (event) => event.loser && event.loser === this
+                    },
+                    message: '{0} is dishonored due to losing a duel',
+                    gameAction: ability.actions.dishonor(context => ({ target: context.source }))
+                })
+            ]
+        });
+        this.action({
+            title: 'Challenge a participating character to a Political duel: the winner draws 2 cards',
```

Comment:
This title is probably too long, it needs to fit nicely on a prompt button.  It doesn't need to give so many details, just to give the player enough to distinguish it from other abilities (e.g. Duelist Training)